api/components/image.schema.json

```diff
@@ -12,16 +12,71 @@
             ]
         },
         "path": {
-            "description": "The image path from the resources folder",
+            "description": "The image path from the resources folder or from the network depending on the value of fromNetwork.",
             "type": "string"
         },
+        "fromNetwork": {
+            "description": "Whether the image will be loaded from a network url of from an application's resource path.",
+            "type": "boolean",
+            "default": false
+        },
         "width": {
-            "description": "The image width",
+            "description": "The image width.",
             "type": "number"
         },
         "height": {
-            "description": "The image height",
+            "description": "The image height.",
             "type": "number"
+        },
+        "alignment": {
+            "description": "How to align the image.",
+            "$ref": "defs/alignment.schema.json"
+        },
+        "centerSlice": {
+            "description": "The center slice for a nine-patch image.",
+            "$ref": "defs/rect.schema.json"
+        },
+        "errorBuilder": {
+            "description": "The error callback when the image encounters an error during loading.",
+            "type": "component"
+        },
+        "excludeFromSemantics": {
+            "description": "Whether to exclude this image from semantics.",
```

Comment:
What is "semantics" ?

Reply:
Semantics is a concept used for accessibility, Flutter has a Semantics tree that contains each widget of the application and its semantic information. Here we have the `excludeFromSemantics` property that is mainly used with images that have no real meaning for the semantics tree. For example, if we use a background image for our application, this might no be important to give a description of it inside the semantics tree.